cinema/models.py

```diff
@@ -84,23 +84,35 @@ class Ticket(models.Model):
     row = models.IntegerField()
     seat = models.IntegerField()
 
-    def clean(self):
+    @staticmethod
+    def validate_ticket(
+            row: int,
+            seat: int,
+            cinema_hall: CinemaHall,
+            errors_to_raise
```

Comment:
The parameter `errors_to_raise` is used in the `validate_ticket` method but is not clearly defined or documented. Consider providing a type hint or documentation to clarify its expected type and usage.